acir/src/circuit/black_box_functions.rs

```diff
@@ -44,6 +45,7 @@ impl BlackBoxFunc {
             BlackBoxFunc::XOR => "xor",
             BlackBoxFunc::RANGE => "range",
             BlackBoxFunc::Keccak256 => "keccak256",
+            BlackBoxFunc::VerifyProof => "verify_proof",
```

Comment:
I changed the name of the `BlackBoxFuncCall` to `RecursiveAggregation` but I was unsure on changing the `BlackBoxFunc` call to that as well. Leaving the builtin with the name `verify_proof` I feel like would still be good but then the mismatch with `BlackBoxFuncCall::RecursiveAggregation` is a bit deceiving.

Reply:
Yeah its somewhat confusing I think, especially since its not verifying a proof